news/5540.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix: `update --outdated` raises NonExistentKey with outdated dev packages
```

Comment:
The linter says:
```
rst ``code`` is two backticks............................................Failed
```
So this should be ``update --outdated``.

Reply:
👍 [3b5558b](https://github.com/pypa/pipenv/pull/5541/commits/3b5558b231457e571f89c7ff323f78ac00efcf94) fixed.